src/mito/src/table.rs

```diff
@@ -560,6 +564,18 @@ impl<R: Region> MitoTable<R> {
         Ok(())
     }
 
+    pub async fn load_region(&self, region_number: RegionNumber, _region: R) -> TableResult<()> {
+        let info = self.table_info.load_full();
+
+        // TODO(weny): support to load the region
```

Comment:
What do we need to do to load a region? Just insert it into the region map in the mito table?

Reply:
Yes, Insert it into the region map.